issueWork/JS1__issue/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script defer>
+    const numArr = [15,52,33,24,75];
```

Comment:
Just a thought: 

I noticed in the standards it asks you to create an array with 5 random numbers twice. One idea you may want to consider in the future is abstracting that into a function to generate the random numbers for you. This way rather than manually type out these numbers for each case, you simply call a function that will handle it for you. If you wanted to take it a step further, you could create a function that takes in a number (n) and generates an array of random numbers with the length of n. Then you can reuse that function here, and any other assignment you have. This will come in handy if you ever need to create large arrays of test data manually.